CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## Unreleased
+
+**New migration patches:** 9
+
+### Fixed
+
+- Fix `set_transaction_id` trigger not using the PK index on `transactions` due to being dependent on `CURRVAL` for every row. Replaced with a CTE the `EXIST` query is now an index-only scan as it was intended to be. As a consequence, scalability of inserts into the `transactions` table has been greatly improved.
```

Comment:
part of me thinks this should more prominently say "MASSIVE speedup, upgrade now!"... other part thinks, "omg don't admit this bug has existed in the codebase for such a long time". Not sure.

Reply:
Transparency is key :D

Reply:
:wave: :tada: @andreasknoepfle check mal campfire gelegentlich :pray: